src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+
+type Props = {
+  filterBy: string,
+  setFilterBy: (value: string) => void
+};
+
+const navigation = ['All', 'Active', 'Completed'];
```

Comment:
there should be `FilterType`s, yo may do it in this way:

```suggestion
const navigation = Object.values(FilterType);
```